apps/admin_app/lib/admin_app_web/router.ex

```diff
@@ -77,6 +77,23 @@ defmodule AdminAppWeb.Router do
     delete("/taxonomy/delete", TaxonomyController, :delete_taxonomy)
     resources("/taxonomy", TaxonomyController, except: [:update])
     post("/taxonomy/create", TaxonomyController, :create_taxonomy)
+
+    get("/products/:product_id/property", ProductController, :index_property)
```

Comment:
Can we use [Nested Resources](https://hexdocs.pm/phoenix/routing.html#nested-resources) here for routes?

Reply:
No we have added all the product property assoc handlers in product controller itself